.github/actions/setup-gopherjs/action.yml

```diff
@@ -0,0 +1,68 @@
+name: Setup GopherJS
+description: Sets up Go, Node.js, and GopherJS
+
+inputs:
+  includeSyscall:
```

Comment:
TBH, I'm entirely happy to drop the node-syscall extension from the CI. It's meed deprecated since GopherJS 1.18, and I was going to remove it in the next stable release entirely.

Reply:
That makes sense. Should we do the same with the `GO111MODULE` tests as well?

Reply:
It may not be relevant for GopherJS users, but there are still (rare) legitimate times when it makes sense to turn off modules, IIRC.  It's a bit unfortunate that the way to do that is with a flag that's ostentiably outdated by 5+ years.

Curious your opinion, @nevkontakte

Reply:
AFAIK, Go team decided to support GOPATH indefinitely, so we have to test both ways too, at least unless we find a really compelling reason to drop it.

Reply:
Cool, that's good to know. I'll move over the TodoMVC GO111MODULE build comparison job too.

Reply:
Added the TodoMVC GO111MODULE build comparison

Reply:
LGTM, thanks!